app.js

```diff
@@ -32,18 +36,22 @@ app.use((err, req, res, next) => {
 
 //PSQL errors - based on err code given by PSQL
 app.use((err, req, res, next) => {
-  switch (err.code) {
-    case "22P02":
-      res
-        .status(400)
-        .send({ status: 400, msg: "400: article_id must be a number" });
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "400: Bad request" });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "23503") {
+    res.status(404).send({ msg: "404: Not found" });
   }
```

Comment:
You could just put these if statements for the different error codes together instead of using separate app.use - nothing you NEED to refactor, but if you want your code to look a bit cleaner you could!